test-etcd-anlyze.sh

```
@@ -0,0 +1,144 @@
+ #!/bin/bash
```

Shellcheck (0.11.0)

[error] Remove leading spaces before the shebang.

(SC1114)